playbooks/common/openshift-cluster/upgrades/pre/verify_control_plane_running.yml

```diff
@@ -9,23 +9,16 @@
       local_facts:
         ha: "{{ groups.oo_masters_to_config | length > 1 }}"
 
-  - name: Ensure Master is running
-    service:
-      name: "{{ openshift.common.service_type }}-master"
-      state: started
-      enabled: yes
-    when: openshift.master.ha is defined and not openshift.master.ha | bool and openshift.common.is_containerized | bool
-
   - name: Ensure HA Master is running
     service:
       name: "{{ openshift.common.service_type }}-master-api"
       state: started
       enabled: yes
-    when: openshift.master.ha is defined and openshift.master.ha | bool and openshift.common.is_containerized | bool
+    when: openshift.common.is_containerized | bool
```

Comment:
Assume we have containerized 3.6 cluster. Before this change the `master` docker service is up and running. There is no `master-api` nor `master-controllers` services running. Once this PR is merged, the check will fail on the cluster as only `master` docker service is running. Well, at least nothing gets updated. Though, that is not what we want, am I right? We should make the check conditional. If the cluster version is 3.6 and less and the deployment is containerized, check what services are actually running/installed and based on that decide which services need to be verified to be running.